resources/wikia/polyfills/bind.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Polyfill for `Function.prototype.bind` because PhantomJS 1.9.8 doens't support it
+ * Should only need to use this for unit tests, as all Wikia's supported browsers support bind.
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind#Polyfill
+ */
+if (!Function.prototype.bind) {
+	Function.prototype.bind = function(oThis) {
```

Comment:
You could get rid of the first if statement and use

Function.prototype.bind = Function.prototype.bind || function(oThis) {

But I honestly can't tell which one is more readable, this way there's just one indentation less :)